mappings/net/minecraft/world/GameRules.mapping

```diff
@@ -1,5 +1,53 @@
 CLASS net/minecraft/class_1928 net/minecraft/world/GameRules
+	FIELD field_19387 Lnet/minecraft/class_1928$class_4313;
+		COMMENT A GameRule which regulates whether fire should spread in a world.
```

Comment:
I think just `Regulates whether fire should spread in a world.` would be better. The field's type already tells us it's a `GameRule`, the comment should just describe what we can't see from its type or name.